examples/oracle/main.tf

```diff
@@ -16,8 +16,8 @@ module "rds_instance_oracle" {
   allocated_storage               = var.allocated_storage
   max_allocated_storage           = var.max_allocated_storage
   subnet_ids                      = local.database_subnets
-  name                            = var.name
-  db_name                         = "oracle"
+  identifier                      = var.identifier
+  db_name                         = "ORACLE"
```

Comment:
just a minor thing: why is this not coming from a variable or local?

Reply:
I've changed it to come from a variable.